docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+
+```
+access_list_create
+access_list_delete
+access_list_member_create
+access_list_member_delete
+access_list_member_update
+access_list_review
+access_list_update
+access_request_create
+access_request_review
+auth
+bot_join
+cert_create
+db_session_query
+db_session_query_failed
+db_session_start
+device_authenticate
+device_enroll
+exec
+instance_join
+join_token_create
+kube_request
+lock_created
+lock_deleted
+recovery_code_used
+reset_password_token_create
+saml_idp_auth
+session_command
+session_join
+session_rejected
+session_start
+user_create
+user_login
+user_password_change
+windows_desktop_session_end
+windows_desktop_session_start
+```
+
+To access the Query Editor, navigate to the `Access Monitoring` section in the Teleport UI and click on the `Query Editor` tab.
+
+
+### Here are a few query examples that demonstrate the capabilities of Access Monitoring
+
+- Query for unique users who executed a ssh command involving `/etc/passwd` file in some way:
+```sql
+SELECT
+  DISTINCT user
+FROM
+  exec
+WHERE
+  command LIKE '%/etc/passwd%';
+```
+
+![exec passwd](../../img/access-monitoring/exec_passwd.png)
+
+- Select the count of unique IP addresses associated with each user cert over different event dates:
+```sql
+SELECT
+  event_date,
+  identity_user AS user,
+  COUNT(DISTINCT cert_create.identity_client_ip) AS unique_ip_count
+FROM
+  cert_create
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date;
+```
+
+![cert ip](../../img/access-monitoring/cert_create_ip.png)
+
+
+- Selects users who interacted with the Teleport cluster from more than one different IP address:
+```sql
+SELECT * FROM (SELECT
+  event_date,
+  identity_user AS user,
+  COUNT(DISTINCT cert_create.identity_client_ip) AS unique_ip_count
+FROM
+  cert_create
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date) AS data WHERE unique_ip_count > 1;
+```
+
+![cert ip count](../../img/access-monitoring/cert_create_ip_count2.png)
+
+- Sow all the unique IP addresses that were used by the user 'admin-annie':
+```sql
+SELECT
+  DISTINCT identity_client_ip
+FROM
+  cert_create
+WHERE identity_user = 'admin-annie'
+```
+![ip user address](../../img/access-monitoring/ip_user_address.png)
+
+- Show access requests and their reviews:
+```sql
+SELECT
+  *
+FROM
+  access_request_create, access_request_review
+WHERE
+  access_request_create.id = access_request_review.id
+```
+
+![access request details](../../img/access-monitoring/access_request_details.png)
+
+
+- Show details about access request and review:
+```sql
+SELECT
+  request.user, request.reason, request.roles, request.resource_ids, review.reviewer, review.state
+FROM
+  access_request_create as request, access_request_review as review
+WHERE
+  request.id = review.id
+```
+![access request](../../img/access-monitoring/access_request.png)
+
+
+### Query Examples with Assist AI
+
+AI assisted query is a feature of the Teleport Access Monitoring Query editor.
```

Comment:
The query editor also integrates with Teleport Assist, making it easy to generate queries with a human-readable description of the data you want to retrieve.